src/Mintpass.sol

```diff
@@ -82,6 +83,15 @@ contract Mintpass is AccessControl, ERC721BBaseTokenURI, ERC721BBurnable, IAutho
         return true;
     }
 
+    /// @dev see {IAuthorizeMints-isValidSignature}
+    /// @dev reverts when signature is not valid
+    /// @param signature must be a bytes value
+    /// @param _tokenURI must be a string value
+    function isValidSignature(bytes memory signature, string memory _tokenURI) external view override returns (bool) {
```

Comment:
this function is fully unrelated to a mintpass. Instead, write a new Authorizer and adapt its interface to what you need. The existing one `function authorizeMint(address minter, address to, bytes memory data) external view returns (bool);` is totally sufficient to hand over signatures to the authorizer (you only must make sure to be able to reconstruct a signed message in the authorizer, but that can be virtually anything)